categories/software-engineering/rules-to-better-code.md

```diff
@@ -90,6 +90,7 @@ index:
   - handle-unhappy-paths
   - get-code-line-metrics
   - single-codebase-for-multiple-domains-with-tinacm-nextjs
+  - do-you-handle-multi-os-dev-teams-in-source-control
```

Comment:
Can you remove the rule from this category and instead add to these two:

- https://www.ssw.com.au/rules/rules-to-better-version-control-with-tfs-aka-source-control/
- https://www.ssw.com.au/rules/rules-to-better-version-control-with-git/

Reply:
✅ Done